engine/src/operators/GroupBy.cpp

```diff
@@ -508,19 +462,18 @@ void aggregationsMerger(std::vector<ral::distribution::NodeColumns> & aggregatio
 }
 
 
-void single_node_aggregations(blazing_frame & input,
+std::unique_ptr<ral::frame::BlazingTable> single_node_aggregations(const ral::frame::BlazingTableView & table,
```

Comment:
this should not be a function distributed and single node should have the same flow, they just get prepared differently